src/gseproject/templates/game/game_home.html

```diff
@@ -6,8 +6,78 @@
 {% block content %}
 
 <button id="button-reset" onclick="resetUserProfile()">Reset Profile</button>
+
+
+
 {% load static %}
+<style>
+    .go-button {
+        position: absolute;
+        bottom: 10%;
+        right: 2%;
+        width: 100px;
+        height: 100px;
+    }
+
+    .pet {
+        position: absolute;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+    }
+</style>
+
 <div class="image-container">
-    <img src="{% static 'images/dragon.png' %}" alt="dragon">
+
+    <div class="pet">
+        <img id="DragonImage" src="{% static 'images/dragon.png' %}" alt="dragon">
+    </div>
+
+    <div class="go-button">
+        <a href="{% url 'shop' %}">
+            <img src="{% static 'images/go.png' %}" alt="execute">
+        </a>
+    </div>
+
 </div>
+
+<script>
+    function petDisappear() {
+        console.log("Pet Disappeared")
+        document.getElementById("DragonImage").style.display = 'none';
+    }
+
+    function petAppear() {
+        document.getElementById("DragonImage").style.display = 'block';
+    }
+
+    document.addEventListener('DOMContentLoaded', function () {
+        petCheckCounter();
+    });
+    function petCheckCounter() {
+        fetch('{% url "pet-check" %}', {
+            method: 'POST',
+            headers: {
+                'X-CSRFToken': getCookie('csrftoken'),
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({}),
+        })
+        .then(data => {
+            // Check if the response JSON is blank (or effectively empty)
+            if (Object.keys(data.json()).length === 0) {
+                console.log('Nothing');
+            // Handle blank response here (e.g., show a message to the user)
+        } else {
+            petAppear();
+            // Handle non-blank response here
+        }
+        })
+            .catch((error) => {
+                console.error('There has been a problem with your fetch operation:', error);
+            });
+    }
+
```

Comment:
I feel these functions are generic enough the be in the template js file, check if there are already (might be under a different name)

Reply:
.go button and .pet are all only in game_home.html